README.md

```diff
@@ -1,2 +1,2 @@
-# chef-server-image cookbook
+# chef_server_image cookbook
```

Comment:
There is a separate PR for Readme: https://github.com/opscode-cookbooks/chef-server-image/pull/18